packages/commercetools/theme/components/AppHeader.vue

```diff
@@ -1,7 +1,8 @@
 <template>
-  <SfHeader 
-    active-sidebar="activeSidebar" 
+  <SfHeader
+    active-sidebar="activeSidebar"
     @click:cart="toggleCartSidebar"
+    @click:account="toggleLoginModalOpen"
```

Comment:
```suggestion
    @click:account="onAccountClicked"
```
You forgot to use your method. btw - how about keeping a convention `onX` for events?

Reply:
Wasnt sure if we want this method or directly do toggle